src/frontend/src/utils/iiConnection.ts

```diff
@@ -511,6 +521,34 @@ export class AuthenticatedConnection extends Connection {
     await actor.remove(this.userNumber, publicKey);
   };
 
+  getIdentityInfo = async (): Promise<
```

Comment:
A client should only use the `getAnchorMetadata`, `setPartialMetadata` and `commitMetadata` functions. `getIdentityInfo` and `setIdentityMetadata` should be `private` then, no?

Reply:
Then the repository won't be able to use it...

I could pass the getter and setter as functions only.